src/application-mapping.ts

```diff
@@ -62,7 +62,8 @@ export function handleApplicationSubmitted(event: ApplicationSubmitted): void {
 	entity.doneReviewerAddresses = []
 	entity.pendingReviewerAddresses = []
 	entity.walletAddress = new Bytes(32)
-
+	entity.claims = mapClaims(applicationId, json.claims)
```

Comment:
Not all JSONs will have a json.claims field (For example - older proposal data). So please handle the case where it is not present.

`if (json.claims) { //Do what you are doing now }`